config/elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    function_naming_convention,
+                    #{regex => "^([a-z]*_?)*(_SUITE)?$"}
```

Comment:
let's change that regex to something that says _"lowercase ascii charts, underscores and digits"_, which is usually the default value for this rule, please